Shaman_22.json

```diff
@@ -0,0 +1,158 @@
+	  {
+        "Name": "Earth Totem",
+        "Key": "7",
+        "Requirement": "MobCount>2",
+        "Cooldown": 30,
+        "Log": false,
+        "MinMana": 30
+      },
+	  {
+        "Name": "Fire Totem",
+        "Key": "F7",
+        "Requirement": "MobCount>2",
+        "Cooldown": 15,
+        "Log": false,
+        "MinMana": 95
+      },
+	   {
+        "Name": "War Stomp",
+        "Key": "0",
+        "Requirements": ["Health%<49","MobCount>2"],
+        "Cooldown": 120,
+        "Log": false,
+        "MinMana": 10
+      },
+      {
+        "Name": "Lightning Bolt",
+        "Key": "2",
+        "StopBeforeCast": true,
+        "HasCastBar": true,
+        "MinMana": 75,
+        "Requirements": ["not InMeleeRange", "Mana%>30"]
+      },
+	  
+      {
+        "Name": "Healing Wave",
+        "Key": "3",
+        "Requirements": ["Health%<45","TargetHealth%>20"],
+		"Cooldown": 5,
+        "HasCastBar": true,
+        "Log": false,
+        "MinMana": 80
+      },
+      {
+        "Name": "AutoAttack",
+        "Requirements": ["not HasRangedWeapon", "not AutoAttacking"]
+      },
+	  {
+        "Name": "Fire shock",
+        "Key": "4",
+        "Requirements": ["Mana%>40", "InMeleeRange"],
+        "Cooldown": 6,
+        "Log": false,
+        "MinMana": 80
+      }
+    ]
+  },
+  "Adhoc": {
+    "Sequence": [
+      {
+        "Name": "Rockbiter Weapon",
+        "StopBeforeCast": false,
+        "Key": "5",
+        "MinMana": 25,
+        "Requirement": "not HasMainHandEnchant"
+      },
+	  {
+        "Name": "Lightning Shield",
+        "StopBeforeCast": false,
+        "Key": "6",
+        "MinMana": 45,
+        "Requirement": "not Lightning Shield"
+    //  },
+	//   {
+    //    "Name": "Well Fed",
+    //    "StopBeforeCast": true,
+    //    "Key": "7",
+    //    "Requirements": ["not Well Fed", "BagItem:6888:1"],
+    //    "Cooldown": 120,
+	//	"DelayAfterCast": 10000,
+    //    "Log": false
+      }
+    ]
+  },
+  "Parallel": {
+    "Sequence": [
+	  {
+        "Name": "Healing Wave",
+		"StopBeforeCast": true,
+        "Key": "3",
+        "Requirement": "Health%<50",
+		"Cooldown": 5,
+        "HasCastBar": true,
+        "Log": false,
+        "MinMana": 80
+      },
+      {
+        "Name": "Food",
+        "HasCastBar": true,
+        "StopBeforeCast": true,
+        "Key": "8",
+        "Requirement": "Health%<40",
+        "Cooldown": 6,
+        "Log": false
+      },
+      {
+        "Name": "Water",
+        "HasCastBar": true,
+        "StopBeforeCast": true,
+        "Key": "9",
+        "Requirement": "Mana%<40",
+        "Cooldown": 6,
+        "Log": false
+      }
+    ]
+  },
+  "NPC": {
+    "Sequence": [
+      {
+        "Name": "Repair",
+        "Key": "R",
+        "Requirement": "Items Broken",
+        "PathFilename": "NPC_Stonetalon.json",
+        "Cost": 6
+      },
+      {
+        "Name": "Sell",
+        "Key": "T",
+        "Requirement": "BagCount>46",
```

Comment:
use bag full instead

Reply:
Oh yes, you know my "bagfull" command cant work until today. I will revise it now.